.github/workflows/cpp.yml

```diff
@@ -271,8 +271,8 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ${{ steps.ccache-info.outputs.cache-dir }}
-          key: cpp-ccache-windows-${{ hashFiles('cpp/**') }}
-          restore-keys: cpp-ccache-windows-
+          key: cpp-ccache-windows-v2-${{ hashFiles('cpp/**') }}
+          restore-keys: cpp-ccache-windows-v2-
```

Comment:
@kou For the record, I had to do this to suppress a weird linker error that I had on this PR but not on my fork.
(perhaps a bug in ccache on Windows?)

Reply:
Good idea.